src/common/fs/watch-file-path.injectable.ts

```diff
@@ -2,12 +2,12 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
-import { apiManager } from "../../../../common/k8s-api/api-manager";
 import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import { watch } from "chokidar";
 
-const apiManagerInjectable = getInjectable({
-  instantiate: () => apiManager,
+const watchFilePathInjectable = getInjectable({
+  instantiate: () => watch,
```

Comment:
This exposes foreign API to our internal modeling. We should wrap this with minimal abstraction. Let's talk about this a bit, otherwise superficial.